src/transform.js

```diff
@@ -0,0 +1,31 @@
+const jscodeshift = require('jscodeshift');
```

Comment:
this is fine but I had wanted to use https://github.com/square/babel-codemod for this.